README.md

```diff
@@ -1,2 +1,7 @@
 # ci-oracle-client-runner
-A container that contains an oracle client for running against Concourse 
+A container that contains an oracle client for running against Concourse.\
```

Comment:
The container isn't bound to Concourse in any way and could for example be used under another CI system that supports containers, so the description here could be more generic, something like: `Docker configuration for Oracle Instant Client.`

Reply:
Yep, fair shout. Updated.